Library/Formula/fmilib.rb

```diff
@@ -0,0 +1,40 @@
+class Fmilib < Formula
+  desc "FMI Library enables integration of Functional Mock-up Units (FMUs)"
+  homepage "http://www.jmodelica.org/FMILibrary"
+  url "http://www.jmodelica.org/downloads/FMIL/FMILibrary-2.0.1-src.zip"
+  sha256 "2b82ad1bbb54399ef595755ca9e12e678e8d1ad0e227609d478d9934a298cc43"
+
+  depends_on "cmake" => :build
+
+  option "with-debug", "Build with debugging symbols"
```

Comment:
Please move this option above the `depends_on` (leave an empty line between them).